src/ansys/mechanical/core/embedding/app.py

```diff
@@ -330,3 +330,73 @@ def update_globals(
     def _update_all_globals(self) -> None:
         for scope in self._updated_scopes:
             scope.update(global_entry_points(self))
+
+    def _print_tree(self, node, max_lines=80, lines_count=0, indentation=""):
+        if lines_count >= max_lines and max_lines != -1:
+            return lines_count
```

Comment:
we should print a note that it is truncating, maybe with a line that says `... truncating after 80 lines`

Reply:
Good idea !